enterprise/frontend/src/embedding-sdk/lib/interactive-question/run-question-on-load.ts

```diff
@@ -35,8 +40,26 @@ export const runQuestionOnLoadSdk =
     const originalQuestion =
       originalCard && new Question(originalCard, metadata);
 
+    let question = new Question(card, metadata);
+
+    question = syncCardParametersWithTemplateTags(question);
+
+    const queryParams = initialSqlParameters
+      ? mapObject(initialSqlParameters, String)
```

Comment:
nit: we usually import the entire underscore
```suggestion
      ? _.mapObject(initialSqlParameters, String)
```

Reply:
Updated in https://github.com/metabase/metabase/pull/50728/commits/d291c7fafbbdbb82b7cedc5291d7f4cff2dc4215

PS. I just noticed we don't have plugins that tree-shakes underscore usage from the bundle, but I suppose that's the least of our worries considering there's so much in the bundle